README.md

```diff
@@ -16,6 +16,7 @@ This is a collection of documents that outline Holiday Extras' culture and devel
  * [Technical Planning Meetings](/technical-planning-meeting.md)
  * [Expedited Procedure](/expedited-procedure.md)
  * [CoffeeScript to JavaScript Guidelines](/coffeescript-to-javascript-guidelines.md)
+ * [Javascript linting Rules](/javascript-linting-rules.md)
```

Comment:
The `/` is redundant: `[Javascript linting Rules](javascript-linting-rules.md)` achieves the same and seems to be the way recommend in the GitHub documentation for relative links.